tests/driver/napi/build.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+use std::io::Write;
+use std::path::PathBuf;
+
+fn os_dylib_prefix_and_suffix() -> (&'static str, &'static str) {
+    if cfg!(target_os = "windows") {
+        ("", "dll")
+    } else if cfg!(target_os = "macos") || cfg!(target_os = "ios") {
+        ("lib", "dylib")
+    } else {
+        ("lib", "so")
+    }
+}
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // target/{debug|release}/build/package/out/ -> target/{debug|release}
+    let mut target_dir = PathBuf::from(std::env::var("OUT_DIR").unwrap());
+    target_dir.pop();
+    target_dir.pop();
+    target_dir.pop();
+
+    let nodejs_native_lib_name = {
+        let (prefix, suffix) = os_dylib_prefix_and_suffix();
+        format!("{}slint_node_native.{}", prefix, suffix)
+    };
+    println!(
+        "cargo:rustc-env=SLINT_NODE_NATIVE_LIB={}",
```

Comment:
Why is this nodejs specific logic included here? The napi generated loader takes care of loading the correct platform-specific file, doesn't it?

Reply:
Removed.